src/containers/Tenant/Diagnostics/Consumers/Consumers.tsx

```diff
@@ -0,0 +1,90 @@
+import {useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import block from 'bem-cn-lite';
+
+import DataTable, {Column} from '@yandex-cloud/react-data-table';
+
+import {DEFAULT_TABLE_SETTINGS} from '../../../../utils/constants';
+import {useAutofetcher} from '../../../../utils/hooks';
+import {Search} from '../../../../components/Search';
+import {getDescribe} from '../../../../store/reducers/describe';
+
+import i18n from './i18n';
+
+import './Consumers.scss';
+
+const b = block('ydb-consumers');
+
+interface Consumer {
+    name: string;
+}
+
+interface ConsumersProps {
+    path: string;
+}
+
+export const Consumers = ({path}: ConsumersProps) => {
+    const dispath = useDispatch();
+
+    const fetchData = () => {
+        dispath(getDescribe({path}));
+    };
+
+    useAutofetcher(fetchData, [path]);
+
+    const data = useSelector((state: any) => state.describe.data);
+    const consumersNames: string[] =
+        data[path]?.PathDescription?.PersQueueGroup.PQTabletConfig.ReadRules || [];
+
+    const consumers: Consumer[] = consumersNames.map((name: string) => ({name}));
+
+    const [consumersToRender, setConsumersToRender] = useState<Consumer[]>(consumers);
```

Comment:
I want to comment on a conceptual issue here

looks like `data`, which is a response from backend, is transformed into some shape for the ui component
in this case `data` is not covered with types, and the final shape is described in this very file

but here is a typing that covers the `data` variable:
https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/types/api/schema.ts#L419
it can and should be used here to restrict the selector return value

Now regarding the `Consumer` interface. Forgive me my verboseness while I'm trying to explain in it a way that makes sense.

it is a dedicated interface, and not just a `string[]`, because in the future there will be more columns in the table — that's good
but when there is more columns, this type extends. The transformation becomes more complex, and most likely needs to be moved to a selector. The selector then needs to have a restricted return value type, which would be this `Consumer` interface. Therefore it should be placed in the folder with store types:
https://github.com/ydb-platform/ydb-embedded-ui/tree/main/src/types/store

all this can as well be done now, both the transformation and the typing